luigi/contrib/kubernetes.py

```diff
@@ -157,18 +157,35 @@ def max_retrials(self):
         """
         return self.kubernetes_config.max_retrials
 
+    @property
+    def delete_on_success(self):
+        """
+        Delete the Kubernetes workload if the job has ended successfully.
+        """
+        return True
+
+    @property
+    def print_pod_logs_on_exit(self):
+        """
+        Fetch and print the pod logs once the job is completed.
+        """
+        return True
```

Comment:
I would have this to False as well. When you run a parallel workload the output would be difficult to read.